bin/start.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+export DB_PORT=27018 # Edit with the wanted port
```

Comment:
Same here.
Eventually a comment that would invite to ensure that the same value is used than in `/bin/install.sh` could be helpful.